core/pom.xml

```diff
@@ -125,12 +125,17 @@
       <artifactId>jetty-servlet</artifactId>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>org.eclipse.jetty</groupId>
+      <artifactId>jetty-servlets</artifactId>
+      <scope>compile</scope>
+    </dependency>
     <!-- Because we mark jetty as provided and shade it, its dependency
```

Comment:
I think this comment needs to be updated (you aren't including orbit here anymore yes?)